packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -15,6 +15,8 @@ export interface MenuItemProps {
   target?: LinkTarget;
   /** Icon of the menu item */
   icon?: IconName;
+  /** Image source if you want to use an image as an icon */
+  imgSrc?: string;
```

Comment:
What is this used for data source logo?

Reply:
<img width="365" alt="Skärmavbild 2021-06-04 kl  15 31 36" src="https://user-images.githubusercontent.com/172951/120809019-f5ad6c80-c549-11eb-9d38-b50c70721419.png">

Reply:
it is a menu when that you can open to be able to change visualizations for the query.

Reply:
We should use a radio-button group instead of hiding it behind an icon,  like we do in alert rule creation 
![Screenshot 2021-06-04 at 16 56 22](https://user-images.githubusercontent.com/5183519/120812599-cc92d900-c555-11eb-985e-ac31316ceb7f.png)

Reply:
@mckn here you go https://www.figma.com/file/2s7EGevGeI4GuiArOmUECK/Unified-Alerting?node-id=1354%3A975